microscopium/screens/cellomics.py

```diff
@@ -29,8 +30,12 @@ def batch_stitch_stack(file_dict, output, order=[0, 1, 2], target_bit_depth=8, *
         files. This dictionary is built using the ``make_key2file`` function.
     output : string
         The directory to output the stitched and concatenated images to.
-    order : list of int, optional
+    stitch_order : array of int, shape (M, N)
+        The order of the stitching.
+        Passed to ``microscopium.cellomics.snail_stitch``
```

Comment:
I think it's module scope *as you would refer* to the linked module, so `..preprocess.stretchlim`. I think. [Here's](http://sphinx-doc.org/domains.html#cross-referencing-python-objects) the clear-as-mud explanation from the documentation...!